landingai/predict.py

```diff
@@ -62,7 +62,7 @@ def _load_api_credential(
         """Create an APICredential object from the provided api_key and api_secret if they are not none.
         Otherwise, it will try to load from the environment variables or .env file.
         """
-        if api_key is None or api_secret is None:
+        if api_key is None:  # api_secret is now optional
```

Comment:
Nice!
we need some documentation along with this change, probably need to explain in what circumstance the api secret is optional, i.e. v1 keys can't be optional. maybe also explain a bit about API key v1 and v2.
I was also thinking maybe it's less error prone and clearer (in terms of naming) if we create a new V2 class for it.

Reply:
Reverted